sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -2118,6 +2129,28 @@ std::pair<RT::PiKernel, std::mutex *> ProgramManager::getOrCreateKernel(
                         &(BuildResult->MBuildResultMutex));
 }
 
+bool doesDevSupportImgAspects(const device &Dev,
+                              const RTDeviceBinaryImage &Img) {
+  const RTDeviceBinaryImage::PropertyRange &PropRange =
+      Img.getDeviceRequirements();
+  return std::all_of(
```

Comment:
No need for all_of

Reply:
Discussed offline - no need for all_of because there is only one "aspects" property in the range.